src/components/RegisterPage/RegisterPage.tsx

```diff
@@ -134,10 +140,7 @@ const RegisterPage = ({ registering, isRegistrationSuccess }) => {
                 )}
               </div>
 
-              <button
-                className={styles.submitBtn}
-                onClick={() => onSubmit}
-              >
+              <button className={styles.submitBtn} onClick={() => onSubmit}>
```

Comment:
forget to call onSubmit function?

Reply:
I don't understand. What should I do with this code?